tableauserverclient/server/endpoint/datasources_endpoint.py

```diff
@@ -262,7 +271,11 @@ def publish(
 
         # Determine if chunking is required (64MB is the limit for single upload method)
         if file_size >= FILESIZE_LIMIT:
-            logger.info("Publishing {0} to server with chunking method (datasource over 64MB)".format(filename))
+            logger.info(
+                "Publishing {} to server with chunking method (datasource over {}MB, chunk size {}MB)".format(
```

Comment:
Are `FILESIZE_LIMIT` and `CHUNK_SIZE` in bytes, not MB?

Reply:
oops, yea